contracts/TopShot.cdc

```diff
@@ -801,6 +802,17 @@ access(all) contract TopShot: NonFungibleToken {
                     return TopShot.resolveContractView(resourceType: nil, viewType: Type<MetadataViews.NFTCollectionDisplay>())
                 case Type<CrossVMMetadataViews.EVMPointer>():
                     return TopShot.resolveContractView(resourceType: nil, viewType: Type<CrossVMMetadataViews.EVMPointer>())
+                case Type<MetadataViews.EVMBridgedMetadata>():
+                    // Project-defined ERC721 EVM contract stores baseURI, name, and symbol in its own contract storage
+                    // Name, symbol, and baseURI below are only used for legacy bridge-deployed ERC721 contract
+                    return MetadataViews.EVMBridgedMetadata(
+                        name: "NBA-Top-Shot",
+                        symbol: "NBAT",
+                        uri: MetadataViews.URI(
```

Comment:
`baseURI` should be set to `https://metadata-api.production.studio-platform.dapperlabs.com/v1/topshot/moment/` on mainnet and left empty on testnet since the staging metadata api isn't exposed publicly